lms/djangoapps/teams/models.py

```diff
@@ -0,0 +1,80 @@
+"""Django models related to teams functionality."""
+
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.translation import ugettext_lazy
+from django_countries.fields import CountryField
+
+from xmodule_django.models import CourseKeyField
+from util.model_utils import generate_unique_readable_id
+from student.models import LanguageField
+
+
+class CourseTeam(models.Model):
+    """This model represents team related info."""
+
+    team_id = models.CharField(max_length=255, unique=True)
+    name = models.CharField(max_length=255)
+    is_active = models.BooleanField(default=True)
+    course_id = CourseKeyField(max_length=255, db_index=True)
+    topic_id = models.CharField(max_length=255, db_index=True, blank=True)
+    date_created = models.DateTimeField(auto_now_add=True)
+    # last_activity is computed through a query
+    description = models.CharField(max_length=300)
+    country = CountryField(blank=True)
+    language = LanguageField(
+        blank=True,
```

Comment:
Isn't the language code required in a `LanguageProficiency`? `UserProfile`s can potentially have many `LanguageProficiency`s, whereas a `CourseTeam` can have exactly one or zero languages. A `UserProfile` represents no proficiency simply by not adding a row to the `LanguageProficiency` table.